src/icons/Times.tsx

```diff
@@ -11,6 +11,10 @@ export interface Props {
   strokeWidth?: number
 }
 
+/**
+ * @deprecated
+ * use src/icons/X.tsx instead
```

Comment:
Times supports animated colors, which I believe we use at least in one place (qr screen headers, when we switch between the tabs). If X should replace this, is this something X should also support?

Reply:
Updated the deprecated note to include a caveat for animated. I don't think we should bring this into the new X icon as it's unclear what support this has in react-native-reanimated v3 - which we should switch to at some point in the future.

Reply:
Removed in 0c069630a7fadb12517810ac4fab7d4693dfb034!